src/main/webapp/WEB-INF/jsp/preview.jsp

```diff
@@ -246,7 +246,7 @@
                     <!-- Email preview table -->
                     <table id="email-list-table" class="list-group" role="list">
                         <tr rsf:id="row:" class="email-row list-group-item" role="listitem">
-                           <td class="right-content-email-toggle">
+                           <td class="right-content-email-toggle" role="presentation">
```

Comment:
> keep the design as is

yes

> Is that on IOS?

Yes on IOS in the native App Context (remember role=application thread...)
in IOS,  `IOS>Settings>accessibility>Set font Size to the maximum`
open App INBOX : email-list isn't displayed with big Size text
so try to two fingers zoom > **no zoom on email-list**

**IOS:**  0  _  **Google INBOX:**  -1

in IOS  `IOS>Settings>accessibility> enable Magnifier`
open App INBOX
in Email-list with the magnifier : a user can read about **30%** of email-list's text, he needs to **scroll on the left and on right** to read 100% of the box.
but when he scroll on the right or the left with the magnifier, **it fires the swipe events right || left**, so _the message is automatically archived || supressed and the message disapears..._

**IOSmagnifier:**  -1  _  **Google INBOX:** -1

> draw 😄